resources/metadata/fortify.yaml

```diff
@@ -0,0 +1,473 @@
+metadata:
+  name: fortifyExecuteScan
+  description: This step executes a Fortify scan on the specified project to perform static code analysis and check the source code for security flaws.
+  longDescription: |-
+    This step executes a Fortify scan on the specified project to perform static code analysis and check the source code for security flaws.
+    
+    The Fortify step triggers a scan locally on your Jenkins within a docker container so finally you have to supply a docker image with a Fortify SCA
+    and Java plus Maven or alternatively Python installed into it for being able to perform any scans.
+spec:
+  inputs:
+    secrets:
+      - name: fortifyCredentialsId
+        description: Jenkins 'Secret text' credentials ID containing token to authenticate to Fortify SSC.
+        type: jenkins
+        aliases:
+          - name: fortifyCredentialId
```

Comment:
Should not be an alias as fortifyCredentialId for the cloud sdk pipeline is pointing to a username password credential and not a token.

Reply:
Thanks!